tests/test_assembleCoadd.py

```diff
@@ -284,6 +284,31 @@ def testDcrGen2Gen3Compatibility(self):
         assembleTask = MockDcrAssembleCoaddTask(config=config)
         self.checkGen2Gen3Compatibility(assembleTask)
 
+    def testOnlineCoaddGen3(self):
+        config = MockInputMapAssembleCoaddConfig()
+        config.statistic = "MEAN"
+        config.validate()
+        assembleTask = MockInputMapAssembleCoaddTask(config=config)
+
+        dataRefList = self.dataRefList
+        results = assembleTask.runQuantum(self.skyInfo, dataRefList)
+        coadd = results.coaddExposure
+
+        configOnline = MockInputMapAssembleCoaddConfig()
+        configOnline.statistic = "MEAN"
+        configOnline.doOnlineForMean = True
+        configOnline.validate()
+        assembleTaskOnline = MockInputMapAssembleCoaddTask(config=configOnline)
+
+        resultsOnline = assembleTaskOnline.runQuantum(self.skyInfo, dataRefList)
+        coaddOnline = resultsOnline.coaddExposure
+
+        self.assertFloatsAlmostEqual(coaddOnline.image.array,
+                                     coadd.image.array, rtol=1e-3)
+        self.assertFloatsAlmostEqual(coaddOnline.variance.array,
+                                     coadd.variance.array, rtol=1e-6)
```

Comment:
Is this the maximum precision at which the images agree? I'd have expected it to be more precise.

Reply:
I _think_ that the difference is because the afw accumulator works with deltas (from the original mean) as opposed to actual summation, but that's not possible with the online version.  But if this level of precision is insufficient, then let me know and I can see if I can make it better.